### PR TITLE
WIP - Issue421 plate view

### LINKED ISSUE #421: BUG - As a user I need to be able to see more of the plate view
**Describe the bug**
When I currently visit the plates view I can only see a partial view and need to scroll to see the rest.

**To Reproduce**
Steps to reproduce the behaviour:
1. ONT > plates
2. show plate
3. I can only see part of a plate

**Expected behaviour**
I would like to see the full plate

**To Reproduce**
Steps to reproduce the behaviour:
1. ONT > runs > create run
2. Drag and drop tube onto flowcell
3. I can only see part of the name 

When I currently visit the new runs page and add a library I can only see part of the name on the flowcell

When I drag a tube onto the flowcell it would be good if the flowcell is highlighted to see where I am dropping it

**Additional context**
This is a CSS issue with bootstrap.

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #421

Changes proposed in this pull request:

* Shows more of plate view. CSS fix, however once pagination is implemented could be reviewed for a better fix since size of table will be known.
* Flowcells now have a white stroke when dropping tube over them. 
* Flowcells input box rotated 90 degrees to be readable.

### CODE REVIEW

File: src/components/ont/OntFlowcell.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
```
Comment (harrietc52) [inline src/components/ont/OntFlowcell.vue:1]: Nice hover bits

File: src/components/ont/OntFlowcell.vue [hunk 2]
```diff
@@ -69,4 +80,22 @@ export default {
   .empty {
     fill: red;
   }
+  .active {
+    stroke: white;
+    stroke-width: 2px;
+  }
+  foreignObject {
```
Comment (harrietc52) [inline src/components/ont/OntFlowcell.vue:87]: What does this bit do? overflow?

File: src/components/ont/OntFlowcell.vue [hunk 3]
```diff
@@ -69,4 +80,22 @@ export default {
...
+    overflow: visible;
+  }
+  input{
```
Comment (harrietc52) [inline src/components/ont/OntFlowcell.vue:90]: This input css seems a little overkill, is it all required?

File: src/components/ont/OntPlate.vue [hunk 4]
```diff
@@ -65,4 +65,9 @@ export default {
   }
 }
 
-</script>
\ No newline at end of file
+</script>
+<style>
```
Comment (harrietc52) [inline src/components/ont/OntPlate.vue:69]: Guessing this is where the magic is! In making the whole plate viewable?

File: src/components/ont/OntFlowcell.vue (hunk 3 above)
Comment (BenTopping) [reply src/components/ont/OntFlowcell.vue:90]: Yes, the transforms allow different browsers to rotate the input box respectively and the physical positioning keeps the input box within flowcell.

**Comment by BenTopping** [comment]
The css for this page appears to be skewed upon looking into it using the browser dev tools. Will work on fixing the whole page's position in this PR

**Comment by harrietc52** [comment]
Yep - working on Firefox now. Happy to approve

**Pull request merged**